src/main/resources/lang/polish.lang

```diff
@@ -0,0 +1,190 @@
+	download in progress: Obecnie trwa pobieranie najnowszej wersji programu Skript.
+	download error: <red>Wystąpił błąd przy pobieraniu najnowszej wersji Skript.:<light red> %s
+	downloaded: Najnowsza wersja programu Skript została pobrana! Uruchom ponownie serwer lub użyj /reload aby zastosować zmiany.
+	internal error: Podczas sprawdzania najnowszej wersji Skript wystąpił wewnętrzny błąd. Szczegółowe informacje znajdują się w logu serwera.
+	custom version: Obecnie używasz niestandardowej wersji Skript. Żadne aktualizacje nie zostaną automatycznie zainstalowane.
+	nightly build: Obecnie używasz wersji rozwojowej programu Skript. Żadne aktualizacje nie zostaną automatycznie zainstalowane.
+
+# -- Commands --
+commands:
+	no permission message: Nie masz wymaganych uprawnień, aby użyć tego polecenia
+	cooldown message: Używasz tego polecenia zbyt często, spróbuj ponownie później
+	executable by players: Ta komenda może być użyta tylko przez graczy
+	executable by console: To polecenie może być użyte tylko przez konsolę
+	correct usage: Prawidłowe użycie:
+	invalid argument: Nieważny argument <gray>'%s<gray>'<reset>. Dozwolone są:
+	too many arguments: To polecenie może zaakceptować tylko jedno %2$s.
+	internal error: Podczas próby wykonania tego polecenia wystąpił wewnętrzny błąd.
+	no player starts with: Nie ma w sieci gracza, którego imię zaczyna się od '%s'
+	multiple players start with: W sieci jest kilku graczy, których nazwiska zaczynają się od '%s'
+
+# -- Hooks --
+hooks:
+	hooked: Udało się zahaczyć o %s
+	error: Nie można było podłączyć się do %1$s. Może się to zdarzyć, jeśli Skript nie obsługuje zainstalowanej wersji %1$s
+
+# -- Aliases --
+aliases:
+	# Errors and warnings
+	empty string: '' nie jest typem pozycji
+	invalid item type: '%s' nie jest typem pozycji
+	empty name: Alias musi mieć nazwę
+	brackets error: Nieprawidłowe użycie nawiasów
+	not enough brackets: Sekcja zaczynająca się od znaku %s ('%s') musi być zamknięta
+	too many brackets: Znak %s ('%s') zamyka nieistniejącą sekcję
+	unknown variation: Zróżnicowanie %s nie został zdefiniowany przed
+	missing aliases: Podążając za Minecraft ids nie mają żadnych aliasów:
+	empty alias: Alias nie ma zdefiniowanego Minecraft id ani tagów.
+	invalid minecraft id: Minecraft id %s jest nieprawidłowy
+	useless variation: Wariacja nie ma określonego Minecraft id ani tagów, więc jest bezużyteczna
+	invalid tags: Podane tagi nie są zdefiniowane w poprawnym JSON.
+	unexpected section: Sekcje nie są tu dozwolone
+	invalid variation section: Sekcja powinna być sekcją wariacyjną, ale %s nie jest prawidłową nazwą wariantu
+	outside section: Aliasy muszą być umieszczone w sekcjach
+
+	# Other messages
+	loaded x aliases from: Załadowano %s angielskie aliasy od %s
+	loaded x aliases: Załadowano łącznie %s angielskie aliasy
+
+# -- Time --
+time:
+	errors:
+		24 hours: Dzień ma tylko 24 godziny
+		12 hours: Użycie formatu 12-godzinnego nie pozwala na więcej niż 12 godzin
```

Comment:
"nie pozwala na więcej niż 12 godzin"
niepełne zdanie, zupełnie bez kontekstu

Reply:
Jest to dobre tłumaczenie